creator-node/src/components/healthCheck/healthCheckComponentService.test.js

```diff
@@ -148,13 +251,6 @@ describe('Test Health Check Verbose', function () {
         highestEnabledReconfigMode: 'RECONFIG_DISABLED'
       }
     }
-    const TranscodingQueueMock = (active = 0, waiting = 0) => {
-      return {
-        getTranscodeQueueJobs: async () => {
-          return { active, waiting }
-        }
-      }
-    }
 
     const res = await healthCheckVerbose(serviceRegistryMock, mockLogger, sequelizeMock, getMonitorsMock, 2, TranscodingQueueMock(4, 0).getTranscodeQueueJobs)
```

Comment:
i know leaving the verbose check in there is temporary until we clean it up in the next phase, but having a test case that explicitly compares the equality of verbose and regular health check would be very reassuring that nothing changed